src/StrictifyFloat.cpp

```diff
@@ -179,8 +120,28 @@ Expr strictify_float(const Expr &e) {
     return Strictify{}.mutate(e);
 }
 
-Expr unstrictify_float(const Expr &e) {
-    return Unstrictify{}.mutate(e);
+Expr unstrictify_float(const Call *op) {
+    if (op->is_intrinsic(Call::strict_add)) {
+        return op->args[0] + op->args[1];
+    } else if (op->is_intrinsic(Call::strict_sub)) {
+        return op->args[0] - op->args[1];
+    } else if (op->is_intrinsic(Call::strict_mul)) {
+        return op->args[0] * op->args[1];
+    } else if (op->is_intrinsic(Call::strict_div)) {
+        return op->args[0] / op->args[1];
+    } else if (op->is_intrinsic(Call::strict_min)) {
+        return min(op->args[0], op->args[1]);
+    } else if (op->is_intrinsic(Call::strict_max)) {
+        return max(op->args[0], op->args[1]);
+    } else if (op->is_intrinsic(Call::strict_lt)) {
+        return op->args[0] < op->args[1];
+    } else if (op->is_intrinsic(Call::strict_le)) {
+        return op->args[0] <= op->args[1];
+    } else if (op->is_intrinsic(Call::strict_eq)) {
+        return op->args[0] == op->args[1];
+    } else {
+        return Expr{};
```

Comment:
I'd add one more `internal_assert(op->is_strict_float_intrinsic())` here to future proof having handled them all?